cluster/config-defaults.yaml

```diff
@@ -907,3 +907,20 @@ apiserver_memory_limit_percent: "80"
 
 # specify if control plane nodes should rely on ASG Lifecycle Hook or not
 control_plane_asg_lifecycle_hook: "true"
+
+# This allows setting custom sysctl settings. The config-item is intended to be
+# used on node-pools rather being set globally.
+#
+# The value is a comma seprated configuration of `sysctl_setting=value` as
+# illustrated in the example below.
+#
+# Example:
+#
+#   sysctl_settings: "fs.aio-max-nr=8388608,fs.inotify.max_user_watches=100000"
+#
+# Which translates to a file on the node:
+#
+#  cat /etc/sysctl.d/01-custom-sysctl-settings.conf
```

Comment:
I am not sure if we would like to put it last.
Like this a default in image will override the settings from the node-pool

Reply:
Changed to `99-` as it was intention to have it override other settings.